modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/js/components/AttributesFields.tsx

```diff
@@ -0,0 +1,179 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {ClayButtonWithIcon} from '@clayui/button';
+import {Option, Picker} from '@clayui/core';
+import ClayForm, {ClayInput} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import ClayLayout from '@clayui/layout';
+import classNames from 'classnames';
+import React, {useState} from 'react';
+
+const booleanValue = [
+	{label: Liferay.Language.get('true'), value: true},
+	{label: Liferay.Language.get('false'), value: false},
+];
+
+const dataTypes = [
+	Liferay.Language.get('string'),
+	Liferay.Language.get('boolean'),
+];
+
+const AttributeFields = ({
+	index,
+	name,
+	onAddClick,
+	onAttributeChange,
+	onRemoveClick,
+	portletNamespace,
+	type,
+	value,
+}) => {
+	const [errorMessage, setErrorMessage] = useState<string>('');
+
+	return (
+		<ClayLayout.Row className="mb-3">
+			<ClayLayout.Col
+				className={classNames({'has-error': errorMessage})}
+				size={4}
+			>
+				<label htmlFor={`${portletNamespace}name_${index}`}>
+					{Liferay.Language.get('attribute')}
+
+					<span className="inline-item-after reference-mark text-warning">
+						<ClayIcon symbol="asterisk" />
+					</span>
+				</label>
+
+				<ClayInput
+					aria-required={true}
+					defaultValue={name}
+					id={`${portletNamespace}name_${index}`}
+					name={`${portletNamespace}name_${index}`}
+					onChange={({target}) => {
+						if (target.value.toLowerCase().trim() === 'src') {
+							setErrorMessage(
+								Liferay.Language.get(
+									'use-the-javaScript-url-field'
+								)
+							);
+						}
+						else {
+							setErrorMessage('');
+
+							onAttributeChange(index, {name: target.value});
+						}
+					}}
+					type="text"
+					value={name}
+				/>
+
+				{errorMessage && (
+					<ClayForm.FeedbackGroup>
+						<ClayForm.FeedbackItem>
+							<ClayForm.FeedbackIndicator symbol="exclamation-full" />
+
+							{errorMessage}
+						</ClayForm.FeedbackItem>
+					</ClayForm.FeedbackGroup>
+				)}
+			</ClayLayout.Col>
+
+			<ClayLayout.Col size={4}>
+				<label htmlFor={`${portletNamespace}type_${index}`}>
+					{Liferay.Language.get('type')}
+				</label>
+
+				<Picker
+					aria-labelledby="picker-label"
+					id={`${portletNamespace}type_${index}`}
+					items={dataTypes}
+					onSelectionChange={(type) =>
+						onAttributeChange(index, {
+							type,
+							value: type !== 'Boolean' ? '' : true,
+						})
+					}
+					selectedKey={type}
+				>
+					{(item) => <Option key={item}>{item}</Option>}
+				</Picker>
+			</ClayLayout.Col>
+
+			<ClayLayout.Col
+				className="d-flex flex-column-reverse justify-content-end"
```

Comment:
I had to make this change because thinking about accessibility, the focus was placed on the plus button first and then on the value input.